src/__experimental__/components/checkbox/checkbox.component.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import tagComponent from '../../../utils/helpers/tags';
+import { validProps } from '../../../utils/ether';
+import FormField from '../form-field';
+import CheckboxStyle from './checkbox.style';
+
+const Checkbox = (props) => {
+  const formFieldProps = () => {
+    const { ...validFormFieldProps } = validProps(
```

Comment:
Why does this need to be scoped in a function? What about something like:
```
const formFieldProps = {
  ...validProps({ propTypes: Checkbox.propTypes, safeProps: ['fieldHelpInline'], props }),
  reverse: !props.reverse
};
```

Reply:
👍